### PR TITLE
Remove CSP referrer policy tests

### PR DESCRIPTION
**Pull request by estark37**
The Referrer Policy spec used to define a 'referrer' CSP directive, but that's gone now, replaced by the Referrer-Policy header.

This pull request removes the generated meta-csp tests that tested the 'referrer' directive. I added one test in referrer-policy/generic/ to test that the 'referrer' directive is NOT supported.

### CODE REVIEW

**Comment by estark37** [comment]
See https://github.com/w3c/webappsec-referrer-policy/issues/57 (cc @jdm)

File: referrer-policy/generic/unsupported-csp-referrer-directive.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```
Comment (jdm) [inline referrer-policy/generic/unsupported-csp-referrer-directive.html]: Is there a .headers file missing from this commit?

**Comment by jdm** [comment]
@estark37 This looks good as long as we sort out where the CSP directive is supposed to come from in the new test.

File: referrer-policy/generic/unsupported-csp-referrer-directive.html (hunk 1 above)
Comment (jdm) [inline referrer-policy/generic/unsupported-csp-referrer-directive.html]: Womp womp. Nevermind me, I can't read.

**Comment by estark37** [comment]
Thanks @jdm!

**Pull request merged**